src/main/resources/reference.conf

```diff
@@ -28,12 +28,15 @@ dataproc {
   #vpcSubnet = "default"
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-08a4ad8"   # updated 8/26
   dataprocDockerImage = "us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da"           # updated 8/26
+  # Set to deploy welder to clusters with the given label
+  #deployWelderLabel = "saturnVersion"
```

Comment:
Commented for now, so we should be able to deploy this prior to deploying the Terra UI changes.

Reply:
what does this label mean?

Reply:
Terra UI sets this label on all clusters (I think the current version is `6`). I'm not sure when they increase it.

Reply:
just want to understand how the label works. How do we get the label on our end?

Reply:
Ah - label is a Leo concept. They're stored in the DB in the `CLUSTER_LABEL` table. Clients can optionally set labels on cluster creation, and use labels to filter `listClusters` results. I happen to know Terra-UI sets this label. :)

Reply:
Gotcha! That makes sense